Migration guide for Keycloak admin client (#20091)
keycloak/keycloak: docs/documentation/release_notes/topics/22_0_0.adoc

In previous releases, the `export` and `import` commands required a `build` command to be run first.
Starting with this release, the `export` and `import` commands perform an automatic rebuild of Keycloak if a build time configuration has changed.

See the migration guide for more details.

> = Renamed Keycloak Admin client artifacts

Reviewer: ```suggestion
= Renamed Keycloak Admin Console artifacts
```

> = Renamed Keycloak Admin client dependencies

Reviewer 2: ```suggestion
= Renamed Keycloak Admin client artifacts
```

> After the upgrade to JakartaEE, dependencies for Keycloak Admin clients were renamed to describe the reality more accurately, with consideration for long-term maintainability.

Reviewer 2: ```suggestion
After the upgrade to JakartaEE, artifacts for Keycloak Admin clients were renamed to describe the reality more accurately, with consideration for long-term maintainability.
```
Maybe we should call it "artifacts"? Not sure about that, but in any case I think dependency is not the right word for what we are distributing.

Author: :grin: Yep, agree. Like, in 5 minutes, I've changed the terminology like 5x when I was considering various points of view. But yeah, 'artifacts' might be better.

> After the upgrade to Jakarta EE, artifacts for Keycloak Admin clients were renamed to describe the reality more accurately, with consideration for long-term maintainability.
> We still provide two separate Keycloak Admin clients, one with Jakarta EE and the other with Java EE support.

Reviewer: ```suggestion
After the upgrade to Jakarta EE, artifacts for Keycloak Admin Console were renamed to more descriptive names with consideration for long-term maintainability.
We still provide two separate Keycloak Admin Consoles, one with Jakarta EE and the other with Java EE support.
```

> After the upgrade to JakartaEE, dependencies for Keycloak Admin clients were renamed to describe the reality more accurately, with consideration for long-term maintainability.
> We still provide two separate Keycloak Admin clients, one with JakartaEE and the other with JavaEE support.

Reviewer 2: ```suggestion
After the upgrade to Jakarta EE, dependencies for Keycloak Admin clients were renamed to describe the reality more accurately, with consideration for long-term maintainability.
We still provide two separate Keycloak Admin clients, one with Jakarta EE and the other with Java EE support.
```
See e.g. https://jakarta.ee/, or [Wiki](https://en.wikipedia.org/wiki/Jakarta_EE).
